gradle.template.properties

```diff
@@ -8,3 +8,7 @@ org.gradle.daemon=false
 # e.g org.gradle.java.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home
 # e.g org.gradle.java.home=C:/Program Files/Java/jdk1.8.0_144
 # org.gradle.java.home=
+
+# Use this property to have the recommended language setting for tests
```

Comment:
I think that will depend on if we keep it commented out or not be default?

I prefer it to be not commented out so that its the default

Reply:
I mean the user doesn't really have any choice do they, the tests are dependent on the locale being minimally an English locale for now?

Reply:
Ohh, sorry misunderstood your statement.


Yeah its not a recommendation but more of a requirement so that tests always pass irrespective of setting of the local computer.
Will change the comment accordingly